proctor-consumer/src/test/java/com/indeed/proctor/integration/SampleSpringApplicationIntegrationTest.java

```diff
@@ -68,7 +72,7 @@ private void runRequestAndCheck(
         final String body = result.getResponse().getContentAsString();
         assertThat(body).isEqualTo(expectedBody);
         assertThat(groupsLogger.getLogFullStringFromAbstractGroups())
```

Comment:
do we need to keep method getLogFullStringFromAbstractGroups?

Reply:
No; it ultimately includes the legacy format. I'll clean this up